extensions/amp-video/0.1/amp-video.js

```diff
@@ -222,9 +225,7 @@ export class AmpVideo extends AMP.BaseElement {
 
     this.video_ = element.ownerDocument.createElement('video');
     // Manage video if the sources contain bitrate or amp-orig-src will be expanded to multiple bitrates.
-    if (
-      this.element.querySelector('source[data-bitrate], source[amp-orig-src]')
-    ) {
+    if (this.isManagedByBitrate_ && !this.isManagedByPool_()) {
```

Comment:
I think we should just remove this code, only Stories use the ABR, let's not create dead paths.

Reply:
Ok SG 👍